### DISCUSSION TITLE
Can't find the saved fit file on android phone

### DISCUSSION THREAD

**Discussion by dstrimble** [cagnulein/qdomyos-zwift#1840] [Q&A]
I looked under /Android folder using CX file browser and nothing is there after clicking the save fit file button in settings on the app.

**Comment by cagnulein** [comment]
@dstrimble are you on android 14? if so join the beta and they are under /documents/qz

**Reply by dstrimble** [reply]
I am. How do I join the beta? Although, this won't get me the file I saved from my ride this morning.

**Reply by cagnulein** [reply]
here https://play.google.com/apps/testing/org.cagnulen.qdomyoszwift

yes qz is copying all the files into the new folder, i didn't tried with files anyway. give it a try, in case i will add also this case

remember that you can add your email address in qz and it will send you a fit file automatically

**Reply by yellowbarchetta** [reply]
btw, I moved out all of the files that had been moved into documents/QZ under Android 14 to tidy things up, but when I reopened the app I think it copied them across all over again from the "hidden" folder into the new one.  Just tried it again and it obviously copies them all across on first opening if they are not there.  Would be nice to allow that action to be suppressed, or the ability to clear the old "dump" of files to tidy things up.

**Comment by cagnulein** [comment]
yes i already thought about this, i will do in the next build

Il giorno gio 30 nov 2023 alle 16:49 yellowbarchetta <
***@***.***> ha scritto:

**Comment by cagnulein** [comment]
done here
https://github.com/cagnulein/qdomyos-zwift/pull/1846

Il giorno gio 30 nov 2023 alle 17:03 cagnulein ***@***.***> ha
scritto:

**Reply by cagnulein** [reply]
@yellowbarchetta  try this https://github.com/cagnulein/qdomyos-zwift/suites/18644086034/artifacts/1084312914
let me know

**Reply by cagnulein** [reply]
@yellowbarchetta Did you try this to check if the move thing works?

**Reply by yellowbarchetta** [reply]
I've lost track!  Can't remember if I've tried this version or not!!

I think the other release which has the same function built in now?  I'm not getting the files moved over any more, so I guess it works?

**Reply by cagnulein** [reply]
I didn't merge this in the master branch so it wasn't included. But if you tried this at least one, the file has been moved and they are not again then.

I will try to merge it so.

**Reply by yellowbarchetta** [reply]
Maybe it was merged into the ,fit  file pace branch?  Or maybe I did try it and I've forgotten!!

**Reply by cagnulein** [reply]
Probably the 2nd :)
